client/main.cpp

```diff
@@ -64,6 +64,8 @@
 
 using namespace std;
 
+PortCustomConfig ahah;
```

Comment:
yeah I'm not comfortable too, but I'm not a developer. I have few basics knowledges in development. I needed to make this work in a little amount of time :/ 

I test with 3 VM, and it work properly, and compil at the same time without errors.

One VM with two scheduler, and the others VM with two daemon running. It's fine and it run without error, with the custom ports setted. But I agree 100% with you, it's not very clean